src/main/java/ml/treecaptcha/uwuify/Uwuify.java

```diff
@@ -12,11 +12,15 @@ public void onEnable() {
         uwu = this;
         getLogger().log(Level.INFO, Uwuifier.uwuify("give uwu sound effects cause idk where to get them"));
         new uwuHandler(this);
-        saveConfig();
+        saveDefaultConfig();
     }
 
     @Override
     public void onDisable() {
         // Plugin shutdown logic
     }
+
+    public static final boolean USE_PREVIEW = uwu.getConfig().getBoolean("use-chat-preview");
+    public static final boolean SIGNS_UWUIFY = uwu.getConfig().getBoolean("signs-uwuify");
+    public static final boolean BOOKS_UWUIFY = uwu.getConfig().getBoolean("books-uwuify");
```

Comment:
This won't work, uwu is set after the static fields are initialized.

Reply:
Crap you're right.